<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitoreo de frenos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Auto.png</normaloff>../../Auto.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="img_Car">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Auto.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Mode">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>22</y>
      <width>55</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BrakePad1">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>176</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Balata es la parte de los frenos que hace fricción para lograr el frenado del automovil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Espesor Balata 1:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="bar_BrakePad1">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>210</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
text-align: right;
margin-right: 21ex;
}
QProgressBar::chunk:horizontal {
background: #00FF00
}</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
    <property name="format">
     <string>%v mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BrakePad1_2">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>176</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Balata es la parte de los frenos que hace fricción para lograr el frenado del automovil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Espesor Balata 2:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="bar_BrakePad2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
text-align: right;
margin-right: 21ex;
}
QProgressBar::chunk:horizontal {
background: #00FF00
}</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
    <property name="format">
     <string>%v mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BrakePad1_4">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>426</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Balata es la parte de los frenos que hace fricción para lograr el frenado del automovil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Espesor Balata 3:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="bar_BrakePad3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
text-align: right;
margin-right: 21ex;
}
QProgressBar::chunk:horizontal {
background: #FF0000
}</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>%v mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BrakePad1_3">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>426</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Balata es la parte de los frenos que hace fricción para lograr el frenado del automovil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Espesor Balata 4:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="bar_BrakePad4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
text-align: right;
margin-right: 21ex;
}
QProgressBar::chunk:horizontal {
background: #00FF00
}</string>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
    <property name="format">
     <string>%v mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SampleRate">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frecuencia de muestreo (seg):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_MinThickness">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>70</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Espesor mínimo aceptable (mm):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Distance1">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>170</y>
      <width>259</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distancia del sensor 1 (mm):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Distance2">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>202</y>
      <width>259</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distancia del sensor 2 (mm):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Distance3">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>236</y>
      <width>259</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distancia del sensor 3 (mm):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Distance4">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>268</y>
      <width>259</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distancia del sensor 4 (mm):</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>737</x>
      <y>130</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Calibracion</string>
    </property>
    <widget class="QLineEdit" name="tb_Distance1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>35</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>150</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_Distance2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>68</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>150</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_Distance3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>101</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>150</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_Distance4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>134</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>150</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Calibracion">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>9</y>
      <width>411</width>
      <height>811</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="tb_SampleRate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>329</x>
       <y>14</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_MinThickness">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>64</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Config">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Configurar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SetConfig">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Cancelar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <zorder>btn_SetConfig</zorder>
    <zorder>tb_SampleRate</zorder>
    <zorder>tb_MinThickness</zorder>
    <zorder>btn_Config</zorder>
    <zorder>btn_Cancelar</zorder>
   </widget>
   <widget class="QComboBox" name="cb_Mode">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Experto</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_BrakeSts">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estado de los frenos:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="lbl_GenStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_MsgSts">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_Status">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>48</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/new/bad/fail.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="gra_Brakes" native="true">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>590</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <zorder>frame1</zorder>
   <zorder>cb_Mode</zorder>
   <zorder>Calibracion</zorder>
   <zorder>groupBox</zorder>
   <zorder>img_Car</zorder>
   <zorder>lbl_Mode</zorder>
   <zorder>lbl_BrakePad1</zorder>
   <zorder>bar_BrakePad1</zorder>
   <zorder>lbl_BrakePad1_2</zorder>
   <zorder>bar_BrakePad2</zorder>
   <zorder>lbl_BrakePad1_4</zorder>
   <zorder>bar_BrakePad3</zorder>
   <zorder>lbl_BrakePad1_3</zorder>
   <zorder>bar_BrakePad4</zorder>
   <zorder>lbl_SampleRate</zorder>
   <zorder>lbl_MinThickness</zorder>
   <zorder>lbl_Distance1</zorder>
   <zorder>lbl_Distance2</zorder>
   <zorder>lbl_Distance3</zorder>
   <zorder>lbl_Distance4</zorder>
   <zorder>lbl_BrakeSts</zorder>
   <zorder>gra_Brakes</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguracion">
    <property name="title">
     <string>Configuracion</string>
    </property>
    <addaction name="actionPuertoSerial"/>
   </widget>
   <addaction name="menuConfiguracion"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPuertoSerial">
   <property name="text">
    <string>PuertoSerial</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_Mode</tabstop>
  <tabstop>tb_SampleRate</tabstop>
  <tabstop>tb_MinThickness</tabstop>
  <tabstop>tb_Distance1</tabstop>
  <tabstop>tb_Distance2</tabstop>
  <tabstop>tb_Distance3</tabstop>
  <tabstop>tb_Distance4</tabstop>
  <tabstop>btn_SetConfig</tabstop>
  <tabstop>btn_Config</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
